lib/internal/Magento/Framework/Locale/TranslatedLists.php

```diff
@@ -215,6 +215,6 @@ public function getCountryTranslation($value, $locale = null)
 
         $translation = (new RegionBundle())->get($locale)['Countries'][$value];
 
-        return $translation ? (string)__($translation) : $translation;
+        return $translation ? (string) $translation : $translation;
```

Comment:
I don't get the point of your change.
Could you provide some Unit Tests to explain the goal of change?